sigh/lib/sigh/resign.rb

```diff
@@ -60,8 +70,11 @@ def resign(ipa, signing_identity, provisioning_profiles, entitlements, version,
 
       puts(command.magenta)
       puts(`#{command}`)
+      command_return_value = $?.to_i
+
+      `security list-keychains -s #{current_keychains}` if current_keychains
```

Comment:
The use of `current` is confusing here. Do you think it makes sense to rename this to `old`, `previous` or `before` or similar?

Reply:
Yes also totally valid in my opinion. I though `current` would be best, but `previous` is also fine.

Reply:
(`current` confused me, as at the time of the second usage it is not the current set of keychains any more, as we changed it...)

Reply:
Yes you are right. Should I change all occurrences of `current_keychains` to `previous_keychains`?

Reply:
Sounds like a plan 👍